src/native-common/Styles.ts

```diff
@@ -20,6 +20,13 @@ const forbiddenProps: string[] = [
     'cursor'
 ];
 
+// React Native styles that ReactXP doesn't expose.
```

Comment:
I'd prefer not to expose these. They needlessly complicate the model and introduce ambiguity if both flex and these are presented within a combined style. They also slow down style adaptation, which is already a measurable source of overhead.